sources/assets/zsh/aliases.d/mobsf

```diff
@@ -0,0 +1 @@
+alias mobsf='cd /opt/tools/Mobile-Security-Framework-MobSF/ && echo "[*] changed dir : cd /opt/tools/Mobile-Security-Framework-MobSF/" && ./run.sh'
```

Comment:
Why we have to go into the directory ?

This doesn't work ?

`alias mobsf='/opt/tools/Mobile-Security-Framework-MobSF/run.sh'`

Reply:
Or would you prefer this: `alias mobsf='poetry --directory /opt/tools/Mobile-Security-Framework-MobSF run mobsf "[::]:8000"'`?